src/aiortc/rtcrtpsender.py

```diff
@@ -232,13 +208,15 @@ async def _handle_rtcp_packet(self, packet):
         if isinstance(packet, (RtcpRrPacket, RtcpSrPacket)):
             for report in filter(lambda x: x.ssrc == self._ssrc, packet.reports):
                 # estimate round-trip time
-                #if self.__lsr == report.lsr and report.dlsr:
-                if report.lsr in self.__lsr_list and report.dlsr:
-                    rtt = time.time() - self.__lsr_time_list[self.__lsr_list.index(report.lsr)] - (report.dlsr / 65536)
+                if self.__lsr == report.lsr and report.dlsr:
+                    self.__log_debug("self.__lsr %s, self.__lsr_time %s", self.__lsr, self.__lsr_time)
```

Comment:
Can you explain exactly what is going on here? I know you mentioned this at the start of the meeting, but I didn't fully follow

Reply:
The `lsr` field in the RTCP packets received from the receiver is the identifier of the RTCP packet and is `lsr = ((self.__ntp_timestamp) >> 16) & 0xFFFFFFFF`. The `ntp_timestamp` at the sender is updated only after sending a new RTP packet. The problem is that we don't have a throughput of 30 fps at this point and the experiment does finish after the sender's video has finished (actually tends to be longer). When there are no more video/lr_video RTP packets to be sent, the `ntp_timestamp` and consequently the `lsr` are not updated, thus the RTCP identifiers are all the same resulting in the rrt numbers being all over the place (the sender does not know which packet is the response of which packets).